src/Display/Slider/Slider.tsx

```diff
@@ -10,15 +10,15 @@ import LeftArrow from './LeftArrow';
 import RightArrow from './RightArrow';
 import SliderItem from './SliderItem';
 
-const Slider = ({
+export const Slider = ({
   afterChange,
   children,
-  autoplay,
+  autoplay = false,
   initialItem,
```

Comment:
Should there be a default value for initialItem?

Reply:
I've thought about this too, but `initialItem` is so weird and messy, I decided not to give one until that's cleaned up.

Reply:
Oh but we're already adding a note about the default value, so it would be good to have it in the table too? Not a major concern though, the purpose is to tell the devs about the default and the note is solving the purpose.

Reply:
I think you misunderstand. There is no default value. The default value is `undefined`, which, when I do `initialItem=undefined`, unfortunately still doesn't show up in the docs table. The description for the param says that the index starts at 1, meaning that if the users choose to control this component, they have to use number from 1 to n-1 to choose which page should be shown.